resources/views/volumes/clone.blade.php

```diff
@@ -6,8 +6,8 @@
     <script type="module">
         biigle.$declare('destinationProjects', {!!$destinationProjects!!});
         biigle.$declare('volume', {!!$volume!!});
-        biigle.$declare('isImageVolume', {{$volume->isImageVolume()}});
-        biigle.$declare('name', '{!!old('name',$volume->name)!!}');
+        biigle.$declare('isImageVolume', {{ $volume->isImageVolume() ? 'true' : 'false' }});
+        biigle.$declare('name', {{ Js::from(old('name', $volume->name)) }});
```

Comment:
This should be enough, as `{{ }}` already escapes strings:

```suggestion
        biigle.$declare('name', '{{ old('name', $volume->name) }}');
```

Reply:
>So {!! Js::from($x) !!} breaks if $x contains a '? This sounds like a bug.

No, I mean when using `{!! old('name', $x) !!}` like before.

> But I'm fine with keeping the `{{ }}` if you think it is safer.

I mainly use the `{{ }}` to avoid errors and because I thought it would prevent xss attacks, but `{!! Js::from() !!}` also works. So then just use `{!! Js::from() !!}` ?

Reply:
Then let's sum up:
- Use `{{ }}` unless it breaks something or the value is an array/object
- Otherwise use `{!! !!}` for arrays/objects and make sure included user-input runs through `escapeHtml()` before it is shown with `v-html`
- Otherwise use `{!! Js::from() !!}` for non-arrays/objects

Correct?

Reply:
Yes!